### PR TITLE
Replace es6-map with a simple implementation of a map.

### PR DESCRIPTION
**Pull request by Mosho1**
#133 

Also, changed the `npm run pragmatist` command so that it works on windows as well.

### CODE REVIEW

File: src/generateAppendClassName.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import Map from 'es6-map';
+import Map from './simple-map';
```
Comment (gajus) [inline src/generateAppendClassName.js]: `simple-map` is very generic. What about calling it `simple-weak-map`?

File: src/simple-map.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+export default class SimpleMap {
```
Comment (gajus) [inline src/simple-map.js]: Please run `npm run lint` and ensure that changes do not introduce new style errors.

File: src/generateAppendClassName.js (hunk 1 above)
Comment (Mosho1) [inline src/generateAppendClassName.js]: I'm not sure that is fitting, as keys are not weakly referenced like in a WeakMap. Your call though.

Comment (benjamingr) [inline src/generateAppendClassName.js]: It's not a weak map. Nor is a weak map required here.

Comment (gajus) [inline src/generateAppendClassName.js]: The reason I have suggested "weak map" is because keys are objects. @benjamingr Can you suggest a better name?

Comment (benjamingr) [inline src/generateAppendClassName.js]: `Map`, in regular ES2015 maps keys can be objects.

A WeakMap is a map that does not interfere with the garbage collection of its keys. If you'd like to see an example of why one would want to use it - see http://stackoverflow.com/questions/29413222/what-are-the-actual-uses-of-es6-weakmap . The "weakness" of the weak map is in that it holds the reference to the keys weakly.

Comment (gajus) [inline src/generateAppendClassName.js]: Thats useful. Thank you. Will keep it as `simple-map` then.

**Comment by benjamingr** [comment]
This should probably fall back to ES2015 maps if they are available.

**Comment by gajus** [comment]
@Mosho1 Thank you for the PR.

Version [v3.7.7](https://github.com/gajus/react-css-modules/tree/v3.7.7) has been released.

**Pull request merged**